### ISSUE TITLE
feature request: Frame Change Threshold and Light Switch Detection fields

### ISSUE THREAD

**Issue by shinji2009** [ccrisan/motioneyeos#795]
hi. 
it is very hard to set Frame Change Threshold and Light Switch Detection to 0.1. it will be nice if such a exact value can be set by typing numbers in the field. actually set to 0.1 like this: set it to 0 and then press Right key on keyboard once, and i think this will be 0.1, but i am not sure because i can't pick it with mouse without move it.
thank you.

**Comment by ccrisan** [comment]
I agree that editing those options can be painful. I'll think of a solution.
